tests/regression/99-tutorials/03-taint_simple.c

```diff
@@ -0,0 +1,40 @@
+// SKIP PARAM: --set "ana.activated[+]" taint --set "exp.extraspecials[+]" readInt --set "exp.extraspecials[+]" printInt  --disable warn.imprecise
+
+int readInt() __attribute__((__taint_source__));
+void printInt(int x,int y ,int z) __attribute__((__taint_sink__));
```

Comment:
There shouldn't be need to add the functions to `extraspecials` because they don't have definitions anyway.

Reply:
Also, why even invent own functions for it? Could just declare additional attributes on `scanf`/`printf`. Or even hardcode them directly in `special`.

Reply:
`scanf` and `printf` would be what one would use for a proper analysis, but here we want the persons taking the tutorial to solve it without having to think of pointers and (even worse) format strings.

Reply:
The format strings are irrelevant though, there's no need to check them or anything. Just consider all arguments sources/sinks.
Alternatively the standard `getchar` and `putchar` could be a simpler choice then.

Reply:
I now use `getchar` and `putchar`, but also left `printInt` to indicate that one can also define custom sinks. Adding to `extraspecials` is more of a cosmetic thing to get rid of ugly warnings about missing implementations (without completely ignoring all `error` messages).

Reply:
`getchar` and `putchar` should have `LibraryFunctions` specifications though, so they shouldn't print missing implementation errors without `extraspecials` either.

Reply:
doesn't look like they do, should I add them here or in a separate PR?

Reply:
Oh, `putchar` does, but `getchar` doesn't. Just add it here, such that the extraspecials wouldn't be needed to hide warnings.